lib/generator.js

```diff
@@ -66,7 +66,6 @@ function buildOperations(spec) {
 
 
       }
-      delete path;
```

Comment:
This operation doesn't seem to have an effect. It also interferes with `nyc` coverage collection (file `generator.js` is not included into coverage report).